WTWSMS/common/traits/02_traits.txt

```diff
@@ -1093,10 +1095,15 @@ schemer = {
 theologian = {
 	lifestyle = yes
 	
-	learning = 2
-	intrigue = 1
+	learning = 4
+	diplomacy = 2
 	
 	ruler_designer_cost = 10
+	church_opinion = 15
+
+	opposites = {
+		cynical
+	}
```

Comment:
I didn't mean cynical of religion altogether, rather cynical of the views of other theologians. Cynical as a trait isn't an extreme atheist in most cases, it can range from a diverse amount of perspectives.